web/src/search/results/SearchResultsFilterBars.tsx

```diff
@@ -31,13 +31,36 @@ export const SearchResultsFilterBars: React.FunctionComponent<{
     calculateShowMoreResultsCount,
 }) => (
     <div className="search-results-filter-bars">
-        {((isSearchResults(results) && filters.length > 0) || extensionFilters) && (
+        {isSearchResults(results) &&
+            results.dynamicFilters.filter(filter => isScopeSelected(navbarSearchQuery, filter.value)).length > 0 && (
+                <div className="search-results-filter-bars__row">
+                    Selected options:
+                    <div className="search-results-filter-bars__filters">
+                        {results.dynamicFilters
+                            .filter(filter => filter.value !== '')
+                            .filter(filter => isScopeSelected(navbarSearchQuery, filter.value))
+                            .map((filter, i) => (
```

Comment:
```suggestion
                            .map(filter => (
```

It doesn't look like `i` is used.